By a Thread/Assets/Scripts/Gameplay/PlayerDeath.cs

```diff
@@ -13,12 +13,14 @@ namespace Platformer.Gameplay
     public class PlayerDeath : Simulation.Event<PlayerDeath>
     {
         PlatformerModel model = Simulation.GetModel<PlatformerModel>();
+        
 
         public override void Execute()
         {
             var player = model.player;
             if (player.health.IsAlive)
             {
+                Debug.Log(player.name);

```

Comment:
It is probably best not to leave debugging logs in when committing, but not worth getting rid of for now

Reply:
I agree - I forgot to remove it, was using it to understand the code